contracts/MerkleDelegation.sol

```diff
@@ -35,34 +31,26 @@ contract MerkleDelegation is Ownable, Pausable {
     constructor(address _governanceToken) Ownable() Pausable() {
         require(
             _governanceToken != address(0),
```

Comment:
maybe out of scope of this PR review but maybe we should consider/debate removing zero address checks? @CjHare 
The check is only necessary for solidity <0.5.7 which we never use? 
https://forum.openzeppelin.com/t/removing-address-0x0-checks-from-openzeppelin-contracts/2222

Reply:
There are two sides to this.

1) Missing argument; as the OZ forum thread cover, omitting an argument is no longer a problem (where a missing argument gets interpreted as the zero address).
2) When the sender passes the zero address as an argument.

What we may wish to consider is using ERC-165 to verify the argument is actually the contract we expect (it does require that contract implement ERC-165).

Reply:
Yes, I think is fine to omit the check as the msg.data cannot be empty now. Regarding ERC-165 I agree is a nice ERC but BIT or most token do not support supportsInterface() to check ERC20 or COMP-like interface.

Reply:
also for delegate it will never be zero due to "_msgSender() == delegator".

Reply:
@aodhgan Slither is forcing the zero-checks in the constructors

`MerkleDelegation.constructor(address)._governanceToken (contracts/MerkleDelegation.sol#30) lacks a zero-check on :
		- governanceToken = _governanceToken (contracts/MerkleDelegation.sol#31)
Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#missing-zero-address-validation`